tsconfig.dist.json

```diff
@@ -15,23 +15,25 @@
     "types": [],
     "typeRoots": [
       "node_modules/@0xproject/types",
-      "node_modules/@0xproject/typescript-typings/types",
+      "node_modules/set-protocol-utils/types",
       "node_modules/@types"
     ],
     "paths": {
       "*": [
         "node_modules/@0xproject/types/*",
-        "node_modules/@0xproject/typescript-typings/types/*",
+        "node_modules/set-protocol-utils/types/*",
         "node_modules/*"
       ]
     },
     "declaration": true,
-    "declarationDir": "./dist/typings"
+    "declarationDir": "./dist/typings",
+    "esModuleInterop": true
```

Comment:
What's the benefit / need for `esModuleInterop`?

Reply:
https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
Required in order to allow for default imports (so we don't have to do `require('...')`) for modules that don't explicitly have that

Reply:
in particular i think web3 and some other package had this problem that this was able to solve